db/migrate/20160525161622_add_label_to_plant_parts.rb

```diff
@@ -0,0 +1,19 @@
+class AddLabelToPlantParts < ActiveRecord::Migration
+  def up
+    add_column :plant_parts, :label, :string, index: true, default: 'CROPSTORE', null: false
+    add_column :plant_parts, :canonical, :boolean, default: false, null: false
+    execute "UPDATE plant_parts SET label = 'PO:0000003' WHERE plant_part = 'whole plant'"
+
+    PlantPart.reset_column_information
+    Rake::Task['obo:plant_parts'].invoke
+  end
+
+  def down
+    remove_column :plant_parts, :label
+    remove_column :plant_parts, :canonical
+
+    PlantPart.all.each do |pp|
+      pp.destroy if pp.plant_scoring_units.count == 0
```

Comment:
This will not work because your branch removes `has_many :plant_scoring_units` from `PlantPart` (you will trigger something like `undefined method `plant_scoring_units' for #<PlantPart:0x00000008916e20>`). Tip: use `column_exists?` to check for the presence of the reference column before trusting ActiveRecord attributes.